docs/submission/clinical-data-validation-rules.md

```diff
@@ -37,19 +37,45 @@ These identifier fields allow for linking across the different clinical events a
 
 ### Time Intervals
 
-To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis.
+To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis. The date of primary diagnosis is the date on which a definitive diagnostic procedure was performed. Validation checks are in place to ensure the values submitted for the different time interval fields make sense according to the following assumptions:
+
+- The `age_at_primary_diagnosis` is used as the reference time point.
+- The day the patient dies is the clinical endpoint (`survival_time`).
+
+Examples of time interval validation checks:
+
+- If a patient's `vital_status` is `Deceased`, all time intervals must be less than or equal to `survival_time`.
+- The `relapse_interval` must be less than the `interval_of_followup` in the Follow Up the relapse was recorded.
```

Comment:
I'm not sure...but maybe can say
The `relapse_interval` must be less than the `interval_of_followup` in the follow up entry that the relapse was recorded.

I just wasn't sure if the end part was complete?

Reply:
Also "Follow Up" isn't capitalized below so doesn't need to be capitalized here